release-notes/topics/aap-212.adoc

```diff
@@ -0,0 +1,58 @@
+* Resolved issues when upgrading automation hub from version 2.1.1 to version 2.1.2
+* Resolved issues where the Controller user interface was not accessible, when automation controller was installed through Operator on OpenShift Core Platform 4.10
+* Fixed an issue that caused the upgrade from Ansible Automation Platform 1.2 to 2.1 to fail, when the execution environment was transferred to automation hub
+* Fixed an issue so that a standalone installation of private automation hub can now transfer the execution environments to the hub
+* Resolved intermittent `500 internal server error` while pushing the execution environments to automation hub that prevents images from being pushed to the hub
+* Fixed an issue when the variable `sslmode` was not configured correctly during automation hub installation
+* Running the ``./setup.sh -b` command from the installer directory no longer fails to load group variables
+* Fixed an issue where the SELinux context was not being configured correctly on subsequent installations
+* The installer no longer fails when IPV6 is disabled
+* Modified the database backup and restore logic to compress the data dump
+* Fixed an issue during installation where a configured proxy would not be used when required
+* Fixed installations of execution environments when installing without internet access
+* Creating default execution environments no longer fails when the password includes special characters
+* Added support to set a parameter to change the temporary directory path, so that the execution environments can be extracted to the bundled installer
+* Upgrading to Ansible Automation Platform 2.1 no longer fails when the Django superuser is missing
+* Fixed an issue where installation of automation hub failed due to symlinks
+* Added explanations for variables of service catalog worker in the installer inventory
+* Fixed an issue where installing automation hub with an external database did not install Postgres as expected
+* Fixed an issue where variables with default values were shown as undefined variables
+* Added support for `ansible_host` in the installer inventory for automation hub
+* Fixed an issue where a few commands were missing on initializing the container
+* Fixed an issue where the TTL controller caused the jobs to run repeatedly
+* Resolved intermittent `500 internal server error` while pulling and pushing the execution environments from private automation hub
+* Fixed an issue so that remotes configured with proxy authentication can be edited without re-entering the proxy password
+* Updated the proxy password from text type to password type to make the password more secure
+* Updated the synchronization that is used by authenticated proxy to avoid authentication failures
+* Added message in the Controller user interface to inform users that the community collections do not have docs
+* Resolved intermittent issue while trying to sync community `devsec.hardening` collection on automation hub 4.4.0
```

Comment:
Does this correspond to https://issues.redhat.com/browse/AAH-1202? If so, it should appear in the 2.1.2 (4.4.4) release notes, according to our [changelog](https://github.com/ansible/galaxy_ng/pull/1386/files)